src/ledger/NetworkConfig.h

```diff
@@ -39,6 +39,8 @@ struct MinimumSorobanNetworkConfig
 
     static constexpr uint32_t MINIMUM_PERSISTENT_ENTRY_LIFETIME = 4'096;
     static constexpr uint32_t MAXIMUM_ENTRY_LIFETIME = 535'680; // 31 days
+
+    static constexpr uint32_t TX_MAX_CONTRACT_EVENTS_SIZE_BYTES = 200;
```

Comment:
Why is the minimum not 0?

Reply:
Because return value is also counted towards events size and upload wasm and create contract ops return the identifiers. I also added a bit more to be able to get at least some basic events working. I guess minimum could be a bit lower (like 40 bytes), but for initial we'd need a bit more to check that events work.

Reply:
Ah that's right. Maybe we should change the name to reflect both events and results? Either way this doesn't need to block the PR.